index.md

```diff
@@ -1,148 +1,49 @@
 ---
 layout: home
 ---
+# 🖥️ Demo
 
-Make Jelly site have a GitBook look!
+All written functions can be tested here Geostyler [Official demo](https://geostyler.github.io/geostyler-demo/) UI. Also checkout [repo](https://github.com/geostyler/geostyler-demo) for the same.
 
-## Demo
+# 🎨 Getting familiar with UI
 
-Live demo on Github Pages: [https://sighingnow.github.io/jekyll-gitbook](https://sighingnow.github.io/jekyll-gitbook)
+[Geostyler](https://geostyler.org/) [Official demo](https://geostyler.github.io/geostyler-demo/) is a very easy to use, interactive web app which allows user to start testing capabilities of the application without installing it. 
 
-[![Jekyll Themes](https://img.shields.io/badge/featured%20on-JekyllThemes-red.svg)](https://jekyll-themes.com/jekyll-gitbook/)
+There are various components to it as follows 
 
-## Why Jekyll with GitBook
+![demo]( images/demo.png "demo")
 
-GitBook is an amazing frontend style to present and organize contents (such as book chapters
-and blogs) on Web. The typical to deploy GitBook at [Github Pages][1]
-is building HTML files locally and then push to Github repository, usually to the `gh-pages`
-branch. It's quite annoying to repeat such workload and make it hard for people do version
-control via git for when there are generated HTML files to be staged in and out.
+## 1. Language
 
-This theme takes style definition out of generated GitBook site and provided the template
-for Jekyll to rendering markdown documents to HTML, thus the whole site can be deployed
-to [Github Pages][1] without generating and uploading HTML bundle every time when there are
-changes to the original repo.
+[Geostyler](https://geostyler.org/) demo UI can be translated in 4 different languages 
```

Comment:
Soon to be 5 :smile:. See https://github.com/geostyler/geostyler/pull/1731

Reply:
done I have added